components/nexusoperations/frontend/handler.go

```diff
@@ -270,6 +270,7 @@ func (h *completionHandler) forwardCompleteOperation(ctx context.Context, r *nex
 		return nexus.HandlerErrorf(nexus.HandlerErrorTypeInternal, "internal error")
 	}
 
+	// TODO: Upgrade Nexus SDK in order to reduce HTTP exposure
```

Comment:
We'll want to modify the Nexus SDK before upgrading it. This is something that doesn't exist yet.

Reply:
I agree -- are you asking for a comment/code change for this PR, or just flagging up and coming work?